packages/mediacenter/kodi/patches/kodi-100.20-remove-texture-packer-requirement.patch

```diff
@@ -0,0 +1,78 @@
+From c6210561c19dac2faba4bbad5162906027b2a0c1 Mon Sep 17 00:00:00 2001
+From: Stefan Saraev <stefan@saraev.ca>
+Date: Wed, 7 Sep 2016 19:47:50 +0300
+Subject: [PATCH 11/11] [cmake] workaround texturepacker
+
+---
+ project/cmake/CMakeLists.txt                     | 7 +------
+ project/cmake/scripts/common/ProjectMacros.cmake | 4 ----
+ project/cmake/scripts/linux/Install.cmake        | 5 -----
+ 3 files changed, 1 insertion(+), 15 deletions(-)
+
+diff --git a/project/cmake/CMakeLists.txt b/project/cmake/CMakeLists.txt
+index c032d92..4c52dc5 100644
+--- a/project/cmake/CMakeLists.txt
++++ b/project/cmake/CMakeLists.txt
+@@ -133,7 +133,6 @@ set(dyload_optional CEC Bluray Plist NFS)
+ set(required_dep_libs EXPAT)
+ 
+ # Required tools
+-find_package(TexturePacker REQUIRED)
+ find_package(JsonSchemaBuilder REQUIRED)
+ 
+ foreach(dep ${required_dep_libs})
+@@ -279,10 +278,6 @@ foreach(skin ${SKINS})
+   copy_skin_to_buildtree(${dir} ${relative})
+ endforeach()
+ 
+-add_custom_target(pack-skins ALL
+-                  DEPENDS TexturePacker::TexturePacker export-files ${XBT_FILES})
+-set_target_properties(pack-skins PROPERTIES FOLDER "Build Utilities")
+-
+ file(MAKE_DIRECTORY ${CMAKE_BINARY_DIR}/system/players/VideoPlayer)
+ 
+ set(LIBCEC_SONAME "${CEC_SONAME}")
+@@ -322,7 +317,7 @@ endif()
+ # main binary
+ if(NOT CORE_SYSTEM_NAME STREQUAL android)
+   add_executable(${APP_NAME_LC} ${CORE_MAIN_SOURCE} "${RESOURCES}" ${OTHER_FILES})
+-  add_dependencies(${APP_NAME_LC} ${APP_NAME_LC}-libraries export-files pack-skins)
++  add_dependencies(${APP_NAME_LC} ${APP_NAME_LC}-libraries export-files)
+ else()
+   set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,--no-undefined")
+   add_library(${APP_NAME_LC} SHARED ${CORE_MAIN_SOURCE} "${RESOURCES}" ${OTHER_FILES})
+diff --git a/project/cmake/scripts/common/ProjectMacros.cmake b/project/cmake/scripts/common/ProjectMacros.cmake
+index 653382c..c6671cf 100644
+--- a/project/cmake/scripts/common/ProjectMacros.cmake
++++ b/project/cmake/scripts/common/ProjectMacros.cmake
+@@ -34,12 +34,8 @@ function(copy_skin_to_buildtree skin)
+   endforeach()
+   file(MAKE_DIRECTORY ${CMAKE_BINARY_DIR}/${dest}/media)
+   string(REPLACE "${CORE_SOURCE_DIR}/" "" dest ${skin})
+-  pack_xbt(${skin}/media ${CMAKE_BINARY_DIR}/${dest}/media/Textures.xbt)
```

Comment:
you need this stuff for skin xbt packing, if you dont want it - you can as well remove texturepacker package.